src/gui/processing/qgsprocessingwidgetwrapperimpl.cpp

```diff
@@ -1267,6 +1267,9 @@ QgsProcessingDurationParameterDefinitionWidget::QgsProcessingDurationParameterDe
   mUnitsCombo->addItem( QgsUnitTypes::toString( QgsUnitTypes::TemporalHours ), QgsUnitTypes::TemporalHours );
   mUnitsCombo->addItem( QgsUnitTypes::toString( QgsUnitTypes::TemporalDays ), QgsUnitTypes::TemporalDays );
   mUnitsCombo->addItem( QgsUnitTypes::toString( QgsUnitTypes::TemporalWeeks ), QgsUnitTypes::TemporalWeeks );
+  mUnitsCombo->addItem( QgsUnitTypes::toString( QgsUnitTypes::TemporalYears ), QgsUnitTypes::TemporalYears );
```

Comment:
I thought we intentionally omitted these because of the ambiguity in year length

Reply:
I was thinking about this again and thought it was still worth adding. For very large intervals, say a decade, its still useful.

Reply:
Hmm - I still disagree. I think it's always going to be misleading, and if we only expose at most "days" then a user will quickly work out for themselves why the other units aren't an option..!